src/DependencyInjection/Configuration.php

```diff
@@ -113,8 +112,8 @@ private function addResettingSection(ArrayNodeDefinition $node): void
                             ->addDefaultsIfNotSet()
                             ->children()
                                 ->scalarNode('template')->cannotBeEmpty()->defaultValue('@SonataUser/Admin/Security/Resetting/email.html.twig')->end()
-                                ->scalarNode('address')->isRequired()->cannotBeEmpty()->end()
```

Comment:
the cannotBeEmpty should be kept, otherwise it allows defining the key with a empty value, right? like the 'template' key.